CRM/Fastactivity/Form/Add.php

```diff
@@ -400,10 +400,10 @@ public function buildQuickForm() {
     }
 
     // Add activity Date Time
-    $this->addDateTime('activity_date_time', ts('Date'), TRUE, array('formatType' => 'activityDateTime'));
+    $this->add('datepicker','activity_date_time', ts('Date'), TRUE, array('formatType' => 'activityDateTime'));
```

Comment:
```suggestion
    $this->add('datepicker','activity_date_time', ts('Date'), array('formatType' => 'activityDateTime'), TRUE);
```
I don't think the above call to add() is right. The signature has the 4th parameter as an attributes array, and that's where you'd put formatType. See
* https://github.com/civicrm/civicrm-core/blob/4d3856c6b71508de8890ac821e73245a1fa1ba0d/CRM/Core/Form.php#L490
* https://github.com/civicrm/civicrm-core/blob/4d3856c6b71508de8890ac821e73245a1fa1ba0d/CRM/Core/Form.php#L529

For activity date you probably want a time component, but to exclude the time component you'd use the 6th parameter to specify `['time' => FALSE]`. See https://github.com/civicrm/civicrm-core/blob/4d3856c6b71508de8890ac821e73245a1fa1ba0d/CRM/Core/Form.php#L546

Example of an older conversion: https://github.com/civicrm/civicrm-core/commit/b59a905f0a708260e92e464a5c5499728ffe0d2c